http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/LeoRoutes.scala

```diff
@@ -145,10 +146,11 @@ class LeoRoutes(
 }
 
 object LeoRoutes {
-  private val clusterNameReg = "([a-z|0-9|-])*".r
+  private val resourceNameReg = "([a-z|0-9|-])*".r
```

Comment:
I think we're slowly deprecating this file...We don't really need to add stuff this file anymore...Can we move the validation to `DisksRoutes`'s companion object?